### DISCUSSION TITLE
맥북 사용자중 유니티에서 빌드했을 때 could not resolve host XXX 에러 해결하신 분 계신가요?

### DISCUSSION THREAD

**Discussion by travelforfreedom** [naverz/zepeto-studio-kor#241] [5. Q&A]
가이드에 명시되어 있는 조건을 충족했음에도 불구하고 첨부한 이미지처럼 에러가 발생해서 다른 해결방법이 있는지 궁금합니다!

1) 20.3.9 버전 사용
2) 스마트폰과 PC 같은 네트워크 사용
- 집에 공유기가 한대뿐이라 와이파이로 양쪽 모두 사용중입니다
3) 방화벽 꺼져 있음
- 맥북이지만 혹시 몰라 확인해봤더니 방화벽이 꺼져 있었습니다

<img width="1280" alt="스크린샷 2022-01-23 오후 4 15 51" src="https://user-images.githubusercontent.com/98252312/150668572-bd9fc3ef-e432-4194-882e-dc6016ff9412.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 최신 월드에서도 같은 현상이 나타나는지 확인 부탁드립니다.
감사합니다.

**Comment by travelforfreedom** [comment]
안녕하세요.
네, 계속 같은 현상이 발생합니다!

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요. 
맥북에서 테스트할 때, QR코드가 나오지 않는 오류 (could not resolve host XXX) 해결방법입니다.

1. 터미널 실행
2. sudo nano /private/etc/hosts 작성후 비밀번호 입력
3. 127.0.0.1 (탭) XXX
(ex. 127.0.0.1 AL01514477.local)
4. ^(Control) + O 를 눌러 저장
5. ^(Control) + X 를 눌러 종료
6. Y 를 눌러 확인
7. sudo dscacheutil -flushcache 입력해서 DNS 초기화

**Reply by almondsky** [reply]
저두 똑같은 오류가 있어서 해결방법을 따라해봤는데,, 아직 해결을 못하고 있는 상황입니다. 뭐가 문제인지 한번만 확인 부탁드릴께요.
1 , 2번 실행후 밑에 캡쳐본은 3번단계 입니다. 
127.0.0.1 (탭을 한 후 ) xxx 를 넣었고 그 후에 저장, 종료 까지 마치고 Y를 누르면 xxx@xxx-iMac ~ % 이런식으로 떠서.. 이게 맞는지 긴가민가 하며.. 마지막 7번을 입력해서 엔터를 쳤습니다. 그리고 다시 실행해보니.. 여전히 같은 문제가 발생하구요.. 
<img width="580" alt="Screen Shot 2022-05-29 at 3 25 36 PM" src="https://user-images.githubusercontent.com/105544091/170855392-a2c85d9d-a6c4-4bc0-8d9b-3320c0886720.png">

**Reply by almondsky** [reply]
추가적으로 Y를 눌렀을때 이렇게 뜨
<img width="368" alt="Screen Shot 2022-05-29 at 3 50 18 PM" src="https://user-images.githubusercontent.com/105544091/170856135-915f367f-50c9-4400-ada1-8971cf9ee170.png">
는데 맞는건가요?

**Reply by almondsky** [reply]
해결했습니다!

**Comment by travelforfreedom** [comment]
해결되었습니다. 감사합니다!
